docs/src/main/sphinx/sql/create-materialized-view.rst

```diff
@@ -0,0 +1,90 @@
+========================
+CREATE MATERIALIZED VIEW
+========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    CREATE [ OR REPLACE ] MATERIALIZED VIEW
+    [ IF NOT EXISTS ] view_name
+    [ COMMENT string ]
+    [ WITH properties ]
+    AS query
+
+Description
+-----------
+
+Create a new materialized view of a :doc:`select` query. The view is a physical
+manifestation of the query results at time of creation. The data is stored and
+can be referenced by future queries. Since the data is stored directly, these
+queries are typically faster than retrieving data from a view created with the
+same query.
+
+However the data in the materialized becomes out of sync with the underlying
```

Comment:
When the data in the source tables referenced in the materialized view query changes, querying the materialized view may either return stale results or the results from its execution as a logical view. This depends on the specific implementation of the connector supporting materialized views.

Reply:
What? Are you saying that a parallel logical view is created?

Reply:
The MV behaves just like a VIEW when the connector tells the engine that it is stale. So a query referencing the MV will just end up running the whole query as if it was referencing a VIEW.

Reply:
I am leaving this out for now and have reworded that section a bit as well